test/functional/feature_notifications.py

```diff
@@ -156,12 +167,43 @@ def run_test(self):
             self.expect_wallet_notify([(bump2, blockheight2, blockhash2), (tx2, -1, UNCONFIRMED_HASH_STRING)])
             assert_equal(self.nodes[1].gettransaction(bump2)["confirmations"], 1)
 
-        # TODO: add test for `-alertnotify` large fork notifications
+        self.log.info("test -alertnotify with large work invalid chain")
+        # create a bunch of invalid blocks
+        tip = self.nodes[0].getbestblockhash()
+        height = self.nodes[0].getblockcount() + 1
+        block_time = self.nodes[0].getblock(tip)['time'] + 1
+
+        invalid_blocks = []
+        for _ in range(7):  # invalid chain must be at least 6 blocks longer to trigger warning
```

Comment:
isn't the "at least" imprecise, and you need _more than_ 6 blocks?
The validation codes uses `>` instead of `>=` and the test fails if I create just 6 block on top of the tip.